src/components/ha-analytics.ts

```diff
@@ -0,0 +1,128 @@
+import "./ha-checkbox";
+import "./ha-settings-row";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { HomeAssistant } from "../types";
+import "./ha-icon-button";
+import { haStyle } from "../resources/styles";
+import type { HaCheckbox } from "./ha-checkbox";
+import { isComponentLoaded } from "../common/config/is_component_loaded";
+import { Analytics } from "../data/analytics";
+
+const PREFERENCES = ["base", "diagnostics", "usage", "statistics"];
+
+@customElement("ha-analytics")
+export class HaAnalytics extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public analytics!: Analytics;
+
+  protected render(): TemplateResult {
+    if (!this.analytics.huuid) {
+      return html``;
+    }
+    return html`
+      <p>
+        ${this.hass.localize(
+          "ui.panel.config.core.section.core.analytics.instance_id",
+          "huuid",
+          this.analytics.huuid
+        )}
+      </p>
+      ${PREFERENCES.map((preference) =>
+        preference === "base" || this.analytics.preferences.includes("base")
```

Comment:
Should we instead of not showing the other options when `base` is not checked, disable them?

Reply:
sure :+1: